common/internal_model/src/main/scala/uk/ac/wellcome/models/parse/PeriodParser.scala

```diff
@@ -26,13 +24,27 @@ object PeriodParser extends Parser[InstantRange] with DateParserUtils {
     "\\[",
     "\\]",
     "\\?",
-    "\\."
-  )
+    "\\.",
+    "\"",
+    "©",
+    // roman numerals are almost always accompanied by a decimal
+    // date, so strip them out if they're at the start of a date
+    // and followed by a word boundary.
+    s"""^"?$romanNumeralRegexString\\b"""
+  ).reduceLeft(_ + "|" + _).r
+
+  // This is from:
+  // https://www.oreilly.com/library/view/regular-expressions-cookbook/9780596802837/ch06s09.html
+  // It is additionally restricted to numerals 3 characters or longer,
+  // to avoid interfering with things like (eg) 'c.1920'", and includes
+  // period and comma separators to handle numerals like "M.DCC.XXXVII."
+  private val romanNumeralRegexString = {
+    val sep = "[\\.\\,]?\\s?"
+    s"(?=[MDCLXVI\\.\\,\\s]{3,})M*$sep(C[MD]|D?C*)$sep(X[CL]|L?X*)$sep(I[XV]|V?I*)".toLowerCase
+  }
```

Comment:
Useful comment, although a quick glance at that page suggests this regex is only for matching valid Roman numerals. Do you know if we have any invalid Roman numerals in the catalogue?

Reply:
I don't, but I had problems with a more permissive regex blatting other things (eg `mid-19th century`). Unscientific but in a quick look through the stuff that couldn't be parsed, I didn't see any Roman numerals...

Reply:
Conclusion: either there aren't any, or they're extremely malformed. 😉